app/assets/javascripts/components/campaign/campaign.jsx

```diff
@@ -29,24 +28,53 @@ export const Campaign = createReactClass({
     if (this.props.campaign.loading) {
       return <div />;
     }
+
+    let campaignHandler;
     if (window.location.href.match(/overview/)) {
-      return <CampaignOverviewHandler {...this.props} />;
+      const countButtons = document.querySelectorAll('button').length;
+      const correction = countButtons > 0 ? `${(countButtons - 1) * 58}px` : '0px';
+      console.log(correction);
+      console.log(countButtons);
+      // const modalClass = document.getElementById('low_modal') ? 'low-modal' : 'low-modal sidebar-correction';
+      campaignHandler = (
+        <div className="high-modal" style={{ marginTop: `${correction}` }}>
+          <CampaignOverviewHandler {...this.props} />
+        </div>
+        );
     }
+
+    if (window.location.href.match(/programs/)) {
+      return (
+        <div>
+          <CampaignNavbar
+            campaign={this.props.campaign}
+          />
+          <header className="main-page">
```

Comment:
Putting this stats in a `header` element is the reason there's extra space above and below the stats on this tab. I think if you remove this, all of the tabs will be consistent.